app/src/main/java/br/stone/mobiletraining/samilasantos/ex02cadastrodocliente/ui/common/Extensions.kt

```diff
@@ -0,0 +1,23 @@
+package br.stone.mobiletraining.samilasantos.ex02cadastrodocliente.ui.common
+
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
+
+fun String.toDate () : Date? = try {
```

Comment:
Você acha okay deixar essas funções de extensão em classes como String (que são usadas para um monte de coisas)?

Como ficaria num app maior com mais funcionalidades?

outra pergunta. Pq vc optou em deixar esse método no pacote de ui e não no domain?

Reply:
Acho interessante deixar na classe String, pois é comum a necessidade de realizar conversão de string para date, uma vez que utiliza-se campos Date e na medida em que o app for evoluindo ter este método pronto agilizaria a implementação. 

Por outro lado o pacote, concordo que faz mais sentido ficar no domínio, farei está atualização para o próximo PR.